counterexamples/addresses/bad-address-white-space.yaml

```diff
@@ -0,0 +1,19 @@
+---
+id: overture:addresses:addres:1
+type: Feature
+geometry:
+  type: Point
+  coordinates: [-71.2086153, 42.3373725]
+properties:
+  theme: addresses
+  type: address
+  update_time: '2024-04-26T23:55:01-08:00'
+  version: 0
+  country: US
+  region: MA
+  city: NEWTON CENTRE
+  postcode: '02459'
+  street: ' COMMONWEALTH AVE'
+  number: '1000'
+  ext_expected_errors:
+    - "[I#/properties/street] does not match pattern [S#/$defs/propertyDefinitions/id/pattern]"
```

Comment:
@jwass can you humour me and update your code editor to add a newline at end of file when saving?

Otherwise you get these "no newline at EOF" warning icons all over the diff.... And one person's touching a file can result in the file going into the diff if you just save it and the save add/deletes a newline.

![image](https://github.com/OvertureMaps/schema/assets/5599909/d746f3ef-c2c8-4635-aa0f-1de8ab4760f7)